stores/sql/main.go

```diff
@@ -2403,7 +2408,18 @@ EXISTS (
 			rhpv2.HostSettings(hs),
 			rhpv3.HostPriceTable(pt),
 		})
+		hostIDs = append(hostIDs, hostID)
 	}
+
+	// fill in v2 addresses
+	err = fillInV2Addresses(ctx, tx, hostIDs, func(i int, addrs []string) {
```

Comment:
Yeah I thought about spending more time on this and being fancy but then again I didn't want to waste too much time on this since we rewrite the db again pretty soon and because I believe the benefit is going to be negligible.
GROUP BY usually results in quite the nasty queries and prepared statement are insanely fast.

Reply:
I'll allow it.